modules/dxp/apps/analytics/analytics-reports-api/src/main/java/com/liferay/analytics/reports/info/action/provider/AnalyticsReportsContentDashboardItemActionProvider.java

```diff
@@ -25,11 +26,43 @@
  */
 public interface AnalyticsReportsContentDashboardItemActionProvider {
 
+	public default ContentDashboardItemAction getContentDashboardItemAction(
+			InfoItemReference infoItemReference,
+			HttpServletRequest httpServletRequest)
+		throws ContentDashboardItemActionException {
+
+		return getContentDashboardItemAction(
+			infoItemReference.getClassName(), infoItemReference.getClassPK(),
+			httpServletRequest);
+	}
+
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), replaced by {@link
+	 *             #getContentDashboardItemAction(InfoItemReference,
+	 *             HttpServletRequest)}
+	 */
+	@Deprecated
 	public ContentDashboardItemAction getContentDashboardItemAction(
 			String className, long classPK,
 			HttpServletRequest httpServletRequest)
 		throws ContentDashboardItemActionException;
 
+	public default boolean isShowContentDashboardItemAction(
+			InfoItemReference infoItemReference,
```

Comment:
Should we follow alphabetical order here?

Reply:
Totally true :)